testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -568,6 +569,136 @@ public void testTrinoWritingDataWithObjectStorageLocationProvider(StorageFormat
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
 
+    private static final String TRINO_INSERTED_PARTITION_VALUES = "" +
+            "(1, 'with-hyphen')," +
+            "(2, 'with.dot')," +
+            "(3, 'with:colon')," +
+            "(4, 'with/slash')," +
+            "(5, 'with\\\\backslashes')," +
+            "(6, 'with\\backslash')," +
+            "(7, 'with=equal')," +
+            "(8, 'with?question')," +
+            "(9, 'with!exclamation')," +
+            "(10, 'with%percent')," +
+            "(11, 'with%%percents')," +
+            "(12, 'with space')";
```

Comment:
can you use same list of values here & in BaseIcebergConnectorTest?
or, is there any reason for them to be different?

Reply:
Sure. The history there was just that Charles wrote the BCT test and Lukasz wrote the Hive product tests, but I combined the interesting BCT cases into the product test.